wlxwzx-JWTT/cloudfunctions/TTSService/config.py

```
@@ -0,0 +1,101 @@
+    'SEGMENT_SEPARATORS': [
+        '。', '！', '？',  # 句号优先
+        '；', '：',        # 分号冒号次之
+        '，', '、',        # 逗号顿号
+        '\n', '\r\n',     # 换行符
+        ' ', '\t'         # 空格制表符
+    ],
+    
+    # 需要清理的字符
+    'CLEANUP_CHARS': [
+        '\r', '\t', '\xa0',  # 特殊空白字符
+        '​', '‌', '‍',         # 零宽字符
```

Pylint (3.3.9)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead.

(E2515)